services/user_service.py

```
@@ -0,0 +1,104 @@
+        user = self.user_repo.find_by_email(email)
+        if not user:
+            raise HTTPException(
+                status_code=status.HTTP_404_NOT_FOUND,
+                detail="유저를 찾을 수 없습니다."
+            )
+        if user.get("profile_image"):
+            self._delete_profile_image_file(user["profile_image"])
+
+        self.user_repo.delete_by_email(email)
+
+    def _delete_profile_image_file(self,file_path:str):
```

Ruff (0.14.11)

Missing return type annotation for private function `_delete_profile_image_file`

Add return type annotation: `None`

(ANN202)